src/plugins/i18n/locales/en.json

```diff
@@ -811,10 +811,10 @@
   "donate": {
     "header": "Donate",
     "content": [
-      "muffon is a non-profit project that has nor ads, neither any paid features or content.",
-      "But, of course, it cannot exist without funds, that are needed for its development and infrastructure.",
-      "A lot of time and effort were put into creating this project, and it would be good if all of the invested would pay off.",
-      "So I ask for your help in muffon support.",
+      "muffon is a non-profit project that has neither ads nor any paid features or content.",
+      "But of course, muffon cannot exist without the funds needed for its development and infrastructure.",
+      "A lot of time and effort were put into creating this project. It would be great if all that investment paid off.",
+      "As such, I would like to ask for your help in supporting muffon.",
       "<strong>Would you like to make a donation?</strong>"
```

Comment:
I also believe changing this to "Please consider making a donation." would sound better. @staniel359 please confirm.